### ISSUE TITLE
first palette color doesn't work on transparent background

### ISSUE THREAD

**Issue by col-one** [aseprite/aseprite#661]
Hi!
If the first palette color is selected, the pen draw nothing. Only when the background is transparent.
I use compiled version from github on linux opensuse 13.1
Thanks.

**Comment by dacap** [comment]
Hi there, you are using Indexed color. The first color palette is the transparent one by default. You can change this index in *Sprite > Properties*, or you can use RGB color mode (You can choose RGB in *File > New*)

**Comment by col-one** [comment]
Okay, thanks sorry for the issue.

**Comment by dacap** [comment]
No problem :wink:

**Comment by imakeretrogames** [comment]
I had the same problem, and found the solution when Google led me here. Is this info in any of the docs?

**Comment by Async0x42** [comment]
Same issue here, googling a few different things finally led me to this answer. Perhaps there should be some sort of UI info describing this in the application? Now that I know, it won't trouble me, but perhaps useful for any new users who aren't familiar with indexed palettes.

**Comment by dacap** [comment]
From some time ago I'm thinking that RGB mode should be the default color mode, as it looks like the expected behavior mode (RGB mode is more like MS Paint or Photoshop, a more user-friendly approach). Nowadays I'm not quite sure. (It's like Indexed mode is related to a specific old-hardware representation which new users don't care about.)

@imakeretrogames there is some information here: http://www.aseprite.org/docs/color#Indexed but it's hard to figure out the relationship between the transparent index, transparent layers, and indexed mode. I'll see how I can improve this aspect (maybe related to #740).

@Async0x42 I think that the Color bar/palette could show the transparent index as "transparent" when a transparent layer is selected, e.g. 
![transparentindex](https://cloud.githubusercontent.com/assets/39654/9157407/7572f3e0-3ed2-11e5-91d6-97e21f2e6f48.png)

Anyway I'm open to new ideas in these aspects. What do you think about RGB being the default color mode?

**Comment by Async0x42** [comment]
RGB as the default was the expected behavior for me at least, when I first used Aseprite, though it was nice to learn about indexed colors and I could see it being useful for certain situations! I definitely like having Indexed Colors there as an option.

I like the idea of making the bottom color bar/palette show as transparent when selected. When I was confused about why the first color was never drawing, I looked a few times at the selection bar in the bottom-left to see if it was transparent, and it wasn't, so I was totally lost.
